src/test/groovy/io/franzbecker/gradle/lombok/LombokPluginIntegrationTest.groovy

```diff
@@ -130,7 +130,7 @@ class LombokPluginIntegrationTest extends AbstractIntegrationTest {
 
         and: "verify the lombok JAR is included"
         def module = new XmlSlurper().parse(imlFile)
-        def provided = module.component.orderEntry.find { it.@scope == "PROVIDED" }
```

Comment:
I made this change because the test was failed and there was no element with `scope="PROVIDED"` in the generated file. But it isn't failed anymore. Strange. You can ignore this change. I will update the PR if you want.

BTW, it's really hard to test the various combination of gradle and IDEs.